setup.py

```diff
@@ -89,7 +89,7 @@
     ],
     extras_require={
         'dev': [
-            'line_profiler',
+            #'line_profiler',
```

Comment:
There's a flake8 error:
`./setup.py:92:13: E265 block comment should start with '# '`